geode-redis/src/main/java/org/apache/geode/redis/internal/data/RedisStringCommandsFunctionExecutor.java

```diff
@@ -63,30 +63,30 @@ public boolean set(ByteArrayWrapper key, ByteArrayWrapper value, SetOptions opti
 
   @Override
   public long incr(ByteArrayWrapper key) {
-    return stripedExecute(key, () -> getRedisString(key).incr(getRegion(), key));
+    return stripedExecute(key, () -> getRedisString(key, false).incr(getRegion(), key));
   }
 
   @Override
   public long decr(ByteArrayWrapper key) {
-    return stripedExecute(key, () -> getRedisString(key).decr(getRegion(), key));
+    return stripedExecute(key, () -> getRedisString(key, false).decr(getRegion(), key));
   }
 
   @Override
   public ByteArrayWrapper getset(ByteArrayWrapper key, ByteArrayWrapper value) {
     return stripedExecute(key,
-        () -> getRedisString(key).getset(getRegion(), key, value));
+        () -> getRedisString(key, false).getset(getRegion(), key, value));
```

Comment:
This is wrong - it should be `true`

Reply:
The story says that we should only update stats for those command that retrieve data.

Reply:
Did you even check what the underlying function does or how native Redis handles this before assuming you knew what do here (or anywhere else)?

We are working on a complex system and need people who pay attention to detail, do their research and not act based on unverified assumptions! This is not a photo sharing app!